executor/executor_test.go

```diff
@@ -2357,6 +2360,32 @@ func (s *testContextOptionSuite) TestCoprocessorPriority(c *C) {
 	cli.mu.Unlock()
 }
 
+func (s *testSuite) TestTimezonePushDown(c *C) {
+	tk := testkit.NewTestKit(c, s.store)
+	tk.MustExec("use test")
+	tk.MustExec("create table t (ts timestamp)")
+	defer tk.MustExec("drop table t")
+	tk.MustExec(`insert into t values ("2018-09-13 10:02:06")`)
+
+	systemTZ := timeutil.Local()
+	c.Assert(systemTZ.String(), Not(Equals), "System")
+	ctx := context.Background()
+	count := 0
+	ctx1 := context.WithValue(ctx, "CheckSelectRequestHook", func(req *kv.Request) {
+		count += 1
+		dagReq := new(tipb.DAGRequest)
+		err := proto.Unmarshal(req.Data, dagReq)
+		c.Assert(err, IsNil)
+		c.Assert(dagReq.GetTimeZoneName(), Equals, systemTZ.String())
+	})
+	tk.Se.Execute(ctx1, `select * from t where ts = "2018-09-13 10:02:06"`)
```

Comment:
Check result ? If tikv/mocktikv has bug, we can cover it.

Reply:
It covers by others test.